pkg/codegen/templates/embedded.tmpl

```diff
@@ -1,37 +1,8 @@
-// Base64 encoded, gzipped, json marshaled Swagger object
-var swaggerSpec = []string{
-{{range .SpecParts}}
-    "{{.}}",{{end}}
-}
-
-// GetSwagger returns the content of the embedded swagger specification file
-// or error if failed to decode
-func decodeSpec() ([]byte, error) {
-    zipped, err := base64.StdEncoding.DecodeString(strings.Join(swaggerSpec, ""))
-    if err != nil {
-        return nil, fmt.Errorf("error base64 decoding spec: %s", err)
-    }
-    zr, err := gzip.NewReader(bytes.NewReader(zipped))
-    if err != nil {
-        return nil, fmt.Errorf("error decompressing spec: %s", err)
-    }
-    var buf bytes.Buffer
-    _, err = buf.ReadFrom(zr)
-    if err != nil {
-        return nil, fmt.Errorf("error decompressing spec: %s", err)
-    }
-
-    return buf.Bytes(), nil
-}
-
-var rawSpec = decodeSpecCached()
+//go:embed spec.yaml
```

Comment:
I think that this will only work if spec.yaml is inside restapi folder 

I think we want to avoid duplicating it

So either we have to embed the spec inside main.go , and provide its content to GetSwagger - either keeping the function in restapi pkg or moving it to go-common